purchases/src/main/kotlin/com/revenuecat/purchases/PostReceiptHelper.kt

```diff
@@ -118,26 +140,66 @@ internal class PostReceiptHelper(
                 storeAppUserID = storeUserID,
                 marketplace = marketplace,
                 onSuccess = { customerInfo, responseBody ->
+                    offlineEntitlementsManager.resetOfflineCustomerInfoCache()
                     subscriberAttributesManager.markAsSynced(
                         appUserID,
                         unsyncedSubscriberAttributesByKey,
                         responseBody.getAttributeErrors()
                     )
                     customerInfoHelper.cacheCustomerInfo(customerInfo)
                     customerInfoHelper.sendUpdatedCustomerInfoToDelegateIfChanged(customerInfo)
-                    onSuccess(customerInfo, responseBody)
+                    onSuccess(customerInfo)
                 },
-                onError = { error, shouldConsumePurchase, isServerError, responseBody ->
-                    if (shouldConsumePurchase) {
+                onError = { error, errorType, responseBody ->
+                    if (errorType == PostReceiptErrorHandlingBehavior.SHOULD_BE_CONSUMED) {
                         subscriberAttributesManager.markAsSynced(
                             appUserID,
                             unsyncedSubscriberAttributesByKey,
                             responseBody.getAttributeErrors()
                         )
                     }
-                    onError(error, shouldConsumePurchase, isServerError, responseBody)
+                    onError(error, errorType, responseBody)
+                }
+            )
+        }
+    }
+
+    private fun useOfflineEntitlementsCustomerInfoIfNeeded(
+        errorType: PostReceiptErrorHandlingBehavior,
```

Comment:
I think this `errorType` should be renamed to something else. It's not representing the type anymore but indicating what the function should be doing

Reply:
Agreed, I missed renaming these... Thanks!